### PR TITLE
[WIP] Add possibility to set admin section access from SSO

### PR DESCRIPTION
**Pull request by zertan**
**What this PR does / why we need it**:

Several customers have requested a possibility to set Member access permissions to different sections upon logging in to 3Scale admin portal using SSO.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-1634
https://issues.redhat.com/browse/THREESCALE-1341
https://issues.redhat.com/browse/THREESCALE-1286
https://issues.redhat.com/browse/THREESCALE-400

**Verification steps** 

1. in SSO add user attribute "admin_sections" containing any of:
   portal, finance, settings, partners, monitoring, plans
   eg. admin_sections: portal##finance##plans
2. add mapper in 3scale-provider-admin client:
   add mapper -> user_attribute -> admin_sections
   enable Multivalued, Add to userinfo
3. login using SSO to admin portal as the user to verify available
   sections match

### CODE REVIEW

**Comment by zertan** [comment]
Hi,

this was the simplest change to make a starting point for this feature. I appreciate all comments and can answer more questions regarding how this functionality is supposed to work.

It is currently a highly sought after changing and have several support cases open from customers:
https://access.redhat.com/support/cases/#/case/02624136

and somewhat related:
https://access.redhat.com/support/cases/#/case/02478445

I would also be happy to join any call to define how we can best design this.

Thank you and best regards,
Daniel

**Comment by zertan** [comment]
@Martouta @HemaHG how shall we proceed? Thanks //Daniel

File: docker-compose.yml [hunk 1]
```diff
@@ -96,5 +98,35 @@ services:
       - "1025:1025"
       - "8025:8025"
 
+  keycloak:
```
Comment (Martouta) [inline docker-compose.yml:101]: I guess that you don't really need this here, right?
Anyway, we have this file outdated and [we plan to remove it soon](https://issues.redhat.com/browse/THREESCALE-3246).

**Comment by Martouta** [comment]
For the next time, better ping @3scale/system instead of individual members of the team 😄 

> I would also be happy to join any call to define how we can best design this.

That's great. Let's do that 👍 

However, those issues are not prioritized for us. So, a meeting to talk about it is fine and probably it will be quicker than speaking about the design here. But you would have to finish it after it unless the priority (order in the queue) changes.

**Comment by hallelujah** [comment]
Sorry, it seems your work did not get enough attention to us.
Thank you for making this project better.

We are going to close it as it was stale for too long.

**Pull request closed without merging**